src/ctap/data_formats.rs

```diff
@@ -176,11 +179,12 @@ pub enum AuthenticatorTransport {
 
 impl From<AuthenticatorTransport> for cbor::Value {
     fn from(transport: AuthenticatorTransport) -> Self {
+        use AuthenticatorTransport::*;
         match transport {
-            AuthenticatorTransport::Usb => "usb",
-            AuthenticatorTransport::Nfc => "nfc",
-            AuthenticatorTransport::Ble => "ble",
-            AuthenticatorTransport::Internal => "internal",
+            Usb => "usb",
+            Nfc => "nfc",
+            Ble => "ble",
+            Internal => "internal",
```

Comment:
I'm not a big fan of this wildcard import. The raw words make it seem like they are identifiers to variables, when in fact they are compile-time constant values. This confused me when implementing the CBOR map destructuring macro.

Also, the type of the enum gets lost.

The import should at the very least be explicit, but in general prefixing enum variants by the enum type is clearer, and more in line with the general Rust style I think.

Reply:
Making these changes is consistent to the implementation of `PublicKeyCredentialSource`, line 530.

What do you mean by

> The import should at the very least be explicit

The import is 2 lines above the match, so I guess this is not what you mean? Does explicit mean to list all variants in the use clause? I'd rather just revert the change then. And also change `PublicKeyCredentialSource` to the new coding style.

Reply:
I meant listing all variants rather than using the wildcard `*` (because with the wildcard it's unclear where the values are coming from).

Reply:
If we list all variants, then there is no benefit to just keeping the old style everywhere. (And also changing all other wildcard uses throughout the file to being explicit.) `PublicKeyCredentialSourceField` uses both styles at the moment.

So if wildcard import is not okay, I vote for removing all `use` clauses entirely.

Reply:
I think removing the `use` clauses entirely would be the most consistent approach (although the most verbose). We have explicit enums for CTAP2 error codes for example.

Reply:
I removed all inner use instructions.